doc/tags/types.rst

```diff
@@ -0,0 +1,30 @@
+``types``
+=======
+
+The ``types`` tag declares the types for one or more names variables.
+
+To do this, specify a :ref:`mapping <twig-expressions>` of names to their types as string constants. 
```

Comment:
```suggestion
To do this, specify a :ref:`mapping <twig-expressions>` of names to their types as strings.
```

Reply:
I wanted to make clear that interpolated strings (let alone concatenated ones) are not supported. But perhaps that's a bit pedantic. Users will run into a `SyntaxError` if they try anything other than constant strings.

Reply:
A concatenation of strings is an expression, not a string. So, "string" is very clear IMHO.